### PR TITLE
Conditional attend button

### PR DESCRIPTION
**Pull request by FluidSense**
This will show a text to users who cannot attend an event for some reason.
It will also display a defective "Meld meg av"-button

### CODE REVIEW

**Review by kharann — APPROVED** [review]
Looks pretty good to me. I do think we maybe should make this a switch case with enums eventually because we have so many cases.

File: src/events/components/DetailView/AttendanceEvent.tsx [hunk 1]
```diff
@@ -23,10 +23,6 @@ const AttendanceEvent: FC<IProps> = ({ eventId }) => {
   // @ts-ignore
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (oleast) [inline src/events/components/DetailView/AttendanceEvent.tsx:24]: Are these still needed?

Comment (FluidSense) [reply src/events/components/DetailView/AttendanceEvent.tsx:24]: Nope! Removing them now!

**Comment by FluidSense** [comment]
> Looks pretty good to me. I do think we maybe should make this a switch case with enums eventually because we have so many cases.
Most cases are handled in OW4 with a status message! Looks way cleaner now imo

**Pull request merged**